scripts/lsync.sh

```diff
@@ -20,7 +20,7 @@ fi
 if [ -d "$1" ] ; then
   SYNCED=1
   for f in `find $1 -name "*.md"` ; do
-    EN_VERSION=`echo $f | sed "s/content\/..\//content\/en\//g"`
+    EN_VERSION=`echo $f | sed "s/content\/.\{2,5\}\//content\/en\//g"`
```

Comment:
I would more prefer this version:
```suggestion
    EN_VERSION=`echo $f | sed 's#content/[^/]*/#content/en/#'`
```

- Change delimiter to `#` to prevent escaping `/` in path.
- In this context, no need to use `"double quote"`.
- The string should only appeared at most once, so remove `/g` (global).
- Use `[^/]` (not slash) instead of `.` (any char).
- We can make sure find result would not have continuous `//`, so `*` instead of `{2,5}` or `+` should be fine.